api/proto/teleport/legacy/types/types.proto

```diff
@@ -3049,6 +3049,9 @@ message AccessRequestSpecV3 {
     (gogoproto.nullable) = true,
     (gogoproto.jsontag) = "expiry,omitempty"
   ];
+
+  // LongTerm is a flag indicating that the request is for long-term access.
+  bool LongTerm = 23 [(gogoproto.jsontag) = "long_term,omitempty"];
```

Comment:
@kiosion Sorry to throw a wrench here but I would prefer we avoid boolean flags to differentiate between "types" or something as previous history shows that it's not flexible enough and sooner or later we realize we need another possible value and end up changing them to enums which comes with painful migrations. Can we use enum here right away e.g. `RequestType` with possible values like `JustInTime` and `LongTerm`?

Reply:
No worries, that makes total sense to me & isn't a huge thing to change either. Will do!